src/Login/LoginMenu.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import helpers from './helpers.js';
```

Comment:
doesn't matter now since there's just the one, but you could also only import the function you need